### PR TITLE
Skip flaky: test_http_endpoint_missing_auth_value

### PR DESCRIPTION
**Pull request by mtojek**
Issue: https://github.com/elastic/beats/issues/30028

This PR skips the flaky test `test_http_endpoint_missing_auth_value`.

### CODE REVIEW

File: x-pack/filebeat/tests/system/test_http_endpoint.py [hunk 1]
```diff
@@ -178,6 +179,7 @@ def test_http_endpoint_wrong_content_header(self):
         assert r.json()[
             'message'] == 'wrong Content-Type header, expecting application/json'
 
+    @unittest.skipIf(os.getenv("CI") is not None and platform.system() == 'Darwin', 'Flaky test: https://github.com/elastic/beats/issues/30028')
```
Comment (jsoriano) [inline x-pack/filebeat/tests/system/test_http_endpoint.py:182]: Nice idea to disable it only in CI.

Comment (jsoriano) [inline x-pack/filebeat/tests/system/test_http_endpoint.py:182]: I think that platform is in the `sys` package.
```suggestion
    @unittest.skipIf(os.getenv("CI") is not None and sys.platform.system() == 'Darwin', 'Flaky test: https://github.com/elastic/beats/issues/30028')
```

Comment (mtojek) [reply x-pack/filebeat/tests/system/test_http_endpoint.py]: Thanks, missed it!

Comment (mtojek) [reply x-pack/filebeat/tests/system/test_http_endpoint.py:183]: Apparently `sys.platform` is not what we're supposed to use. It's `platform` :)

**Comment by rdner** [comment]
@mergifyio update

**Comment by rdner** [comment]
@jsoriano could you take a final look on this?

**Review by jsoriano — APPROVED** [review]

**Pull request merged**